docs/conf.py

```diff
@@ -65,7 +65,6 @@
     "sphinx.ext.napoleon",
     "sphinx.ext.viewcode",
     "sphinx_copybutton",
-    "sphinx_panels",
```

Comment:
Please note that I removed this package because I wanted to update sphinx to 5.0.0 since:  
- We were seeing this error in the docs build pipeline (see [link](https://readthedocs.org/projects/gpt-engineer/builds/23230027/)). 
- This package, which was not being uses anywhere, is not maintained anymore ([link](https://sphinx-panels.readthedocs.io/en/latest/)) and impeding us to update to this  more recent version of sphinx.